cypress/component/StudyAssets/funding_resource_list.spec.tsx

```diff
@@ -24,15 +24,21 @@ const FundingResourceListHarness: React.FC<{ initial: FundingResource[] }> = ({
   return (
     <FundingResourceList
       fundingResources={items}
-      columnsToShow={['funderName']}
+      columnsToShow={['funderName', 'funderProgram']}
```

Comment:
I was wondering about this in my recent PR. Is there a better way to reference interface fields other than as plain strings?

Reply:
I will introduce stronger typing to these columns such as

``
  readonly columnsToShow?: (keyof FundingResource)[]
``